RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -76,6 +73,16 @@ import RevenueCat
         }
     }
 
+    var hasPurchases: Bool {
+        !purchasesActive.isEmpty || purchaseInformation != nil
+    }
+
+    @Published
+    var purchasesActive: [PurchaseInformation] = []
+
+    @Published
+    var purchaseInformation: PurchaseInformation?
```

Comment:
hmmm is this used anymore?

Reply:
This is used when there's only an active non-subscription. Once we display everything on the main screen, I'll get rid of it